packages/frontend/src/app/modules/flow-builder/page/flow-builder/flow-right-sidebar/edit-step-sidebar/edit-step-accordion/input-forms/piece-input-forms/piece-trigger-input-form/piece-trigger-input-form.component.ts

```diff
@@ -137,7 +135,7 @@ export class PieceTriggerInputFormComponent {
           const configs = Object.entries(trigger.props).map((keyEntry) => {
```

Comment:
can you please make keyEntry => [propName,prop] ? sorry I should have done that at the start.

Reply:
fixed in 6ce6500